src/app/Http/Requests/VerifyRequest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class VerifyRequest extends FormRequest
+{
+    public function authorize()
+    {
+        return true;
+    }
+
+    public function rules()
+    {
+        return [];
+    }
+}
```

Comment:
Why empty??

Reply:
All users are authorized to make the request po sir and the rules is empty because no specific validation rules for the data in this request

Reply:
if no rules for validation requests why did you create this??

Reply:
I deleted this verify request sir and edit my verificationcontroller, Thank you!